data/strings/strings.txt

```diff
@@ -22172,220 +22172,177 @@
-    es-MX = Cruces de ferri
-    et = Praamiületused
-    eu = Ferry pasabideak
-    fa = گذرگاه‌های جاده‌ای
-    fi = Lauttaliikenne
-    fr = Traversées en ferry
-    hu = Kompátkelők
-    id = Penyeberangan kapal feri
-    it = Traghetti
-    ja = フェリー航路
-    ko = 여객선 교차로
-    lt = Kelionė keltu
-    mr = जल मार्ग
-    nb = Fergeoverganger
-    nl = Veerdiensten
-    pl = Przeprawy promowe
-    pt = Ferry
-    pt-BR = Balsa
-    ro = Treceri cu bac
-    ru = Паромные переправы
-    sk = Prechody trajektom
-    sv = Färjetrafik
-    sw = Vivuko cha feri
-    th = เรือข้ามฟาก
-    tr = Feribot geçişleri
-    uk = Поромні переправи
-    vi = Bến phà
-    zh-Hans = 渡轮渡口
-    zh-Hant = 渡輪渡口
+    en = Avoid ferry crossings
+    af = Vermy veerbootoorgange
+    ar = تجنب معابر العبارات
+    be = Пазбягайце паромных пераправаў
+    bg = Избягвайте преминаването с ферибот
+    ca = Eviteu les travessies en ferri
+    cs = Vyhněte se přejezdům trajektů
+    da = Undgå færgeoverfarter
+    de = Fähren vermeiden
+    el = Αποφύγετε τις διελεύσεις με φέρι
+    es = Evitar ferris
+    es-MX = Evitar ferris
+    et = Vältige praamiületusi
+    eu = Saihestu ferry-pasaketak
+    fa = از عبور کشتی ها خودداری کنید
+    fi = Vältä lauttareittejä
+    fr = Éviter les traversées en ferry
+    hu = Kerülje a kompátkelőhelyeket
+    id = Hindari penyeberangan feri
+    it = Evitare le traversate in traghetto
+    ja = フェリーの横断を避ける
+    ko = 페리 횡단을 피하십시오
+    lt = Venkite keltų perėjų
+    mr = फेरी क्रॉसिंग टाळा
+    nb = Unngå fergeoverfarter
+    nl = Vermijd veerbootovertochten
+    pl = Unikaj przepraw promowych
+    pt = Evitar balsas
+    pt-BR = Evitar balsas
+    ro = Evitați traversările cu feribotul
+    ru = Избегать паромных переправ
+    sk = Vyhnite sa trajektovým prechodom
+    sv = Izogibajte se prehodom s trajektom
+    sw = Undvik färjeöverfarter
+    th = หลีกเลี่ยงการข้ามเรือข้ามฟาก
+    tr = Feribot geçişlerinden kaçının
+    uk = Уникати поромних переправ
+    vi = Tránh qua phà
+    zh-Hans = 避免渡口
+    zh-Hant = 避免渡口
 
   [avoid_motorways]
     tags = android,ios
-    en = Motorways
-    af = Snelweg
-    ar = الطرق السريعة
-    be = Аўтамагістралі
-    bg = Магистрали
-    ca = Autopistes
-    cs = Dálnice
-    da = Motorveje
-    de = Autobahnen
-    el = Αυτοκινητόδρομοι
-    es = Autopistas
-    es-MX = Autopistas
-    et = Kiirteed
-    eu = Autobideak
-    fa = بزرگراه‌ها
-    fi = Valtatiet
-    fr = Autoroutes
-    hu = Autópályák
-    id = Jalur motor
-    it = Autostrade
-    ja = 高速道路
-    ko = 고속도로
-    lt = Automagistralės
-    mr = महामार्ग
-    nb = Motorvei
-    nl = Autosnelwegen
-    pl = Autostrady
-    pt = Autoestradas
-    pt-BR = Autoestradas
-    ro = Autostrăzi
-    ru = Магистрали
-    sk = Diaľnice
-    sv = Motorväg
-    sw = Barabara za mwendo kasi
-    th = ทางด่วน
-    tr = Otoyollar
-    uk = Магістралі
-    vi = Đường cao tốc
-    zh-Hans = 高速公路
-    zh-Hant = 高速公路
+    en = Avoid motorways
```

Comment:
Also note different us and uk translations.

```
    en = Avoid freeways
    ar = تجنب الطريق السريع
    be = Пазбягаць аўтамагістралі
    bg = Избягвайте магистралата
    ca = Eviteu l'autopista
    cs = Vyhněte se dálnici
    da = Undgå motorvejen
    de = Vermeide die autobahn
    el = Αποφύγετε τον αυτοκινητόδρομο
    en-GB = Avoid motorways
    es = Evita la autopista
    es-MX = Evite la autopista
    et = Vältige kiirteed
    eu = Saihestu autobidea
    fa = از بزرگراه دوری کنید
    fi = Vältä moottoritietä
    fr = Évite l'autoroute
    he = הימנע מהכביש המהיר
    hi = मोटरमार्ग से बचें
    hu = Kerülje az autópályát
    id = Hindari jalan raya
    it = Evita l'autostrada
    ja = 高速道路を避ける
    ko = 고속도로를 피하세요
    mr = मोटरवे टाळा
    nb = Unngå motorveien
    nl = Vermijd de snelweg
    pl = Unikaj autostrad
    pt = Evita a autoestrada
    pt-BR = Evite a rodovia
    ro = Evitați autostrada
    ru = Избегать автомагистрали
    sk = Vyhnite sa diaľnici
    sv = Undvik motorvägen
    sw = Epuka barabara
    th = หลีกเลี่ยงมอเตอร์เวย์
    tr = Otoyoldan kaçının
    uk = Уникати автомагістралі
    vi = Tránh xa lộ
    zh-Hans = 避开高速公路
    zh-Hant = 避開高速公路
```

Reply:
+1 @biodranik please, fix the french translation as initially suggested.

Reply:
+1 Please do not rewrite PT and PT-BR strings, since they were already revised.

Reply:
Don't worry, I will keep all the revised string the same.